source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,87 @@
+.. _php-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: setup, composer, installation, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure that you have the following
+      dependencies installed on your local machine:
+      
+      - `PHP <https://www.php.net/manual/en/install.php>`__
+      - `Composer <https://getcomposer.org/download/>`__
+    
+   .. step:: Install the MongoDB extension
+
+      Run the following command to install the ``mongodb`` PHP extension:
+      
+      .. code-block:: bash
+
+         sudo pecl install mongodb
+
+   .. step:: Update your PHP configuration file
+
+      To enable the ``mongodb`` extension, add the following line to the top of
+      your ``php.ini`` file:
+
+      .. code-block:: bash
+
+         extension=mongodb.so
+
+      .. tip::
+
+         You can locate your ``php.ini`` file by running the following command
+         from your shell:
+
+         .. code-block:: bash
+
+            php --ini
+
+   .. step:: Create a project directory
+
+      From your root directory, run the following command in your shell to create
+      a directory called ``php-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir php-quickstart
+
+      Run the following commands to create a ``quickstart.php`` application file in
+      the ``php-quickstart`` directory:
+
+      .. code-block:: bash
+
+         cd php-quickstart
+         touch quickstart.php
```

Comment:
S: To my knowledge, the touch command only works on linux/mac machines. This thread: https://stackoverflow.com/questions/30011267/create-an-empty-file-on-the-commandline-in-windows-like-the-linux-touch-command provides alternatives that I think work on both Windows and Linux, unless we want to specify that this guide is only valid for mac/linux machines